### PR TITLE
add clarification that links should be included in snapshot polls

### PR DESCRIPTION
**Pull request by shelbeeee**

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ The _Formal Review_ phase serves as the final review step in the governance proc
...
 - If applicable, inlcude a draft of the executable proposal code for the community to review. See this template [INSERT LINK WHEN READY] for how the code should be presented. 
-- Publish a corresponding Snapshot poll to your proposal. To initiate a Snapshot poll for Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. Proposal authors should include a link to the Snapshot poll at the top of their proposal draft on the forum.
+- Publish a corresponding Snapshot poll to your proposal. To initiate a Snapshot poll for Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal.
+- At the top of the Snapshot poll, please include a links to both the Discussion and Formal Review forum posts.
```
Comment (abbey-titcomb) [inline manual.md]:
```suggestion
- At the top of the Snapshot poll, please include links to both the Discussion and Formal Review forum posts.
```

**Review by abbey-titcomb — CHANGES_REQUESTED** [review]
fixed one grammatical error!

**Review by abbey-titcomb — APPROVED** [review]
lgtm!

**Comment by shelbeeee** [comment]
creating a new pr to fix conflicts

**Pull request closed without merging**